assets/data/cards_pt-br.json

```
@@ -571,10 +571,8 @@
     },
     {
       "title": "Polimorfismo",
-      "tags": [
-        "Conceito"
-      ],
-      "description": "O Polimorfismo permite que objetos de diferentes classes sejam tratados uniformemente, cada um respondendo à sua maneira de acordo com uma interface. Exemplo: Podemos em um sistema ter a interface de funcionário onde temos um método calcularSalario, e está interface tem 3 classes que implementam ela, sendo elas: Analista, Gerente e Diretor. Cada uma dessas classes implementa o método calcularSalario de uma forma diferente."
+      "tags": ["Conceito"],
+      "description": "O polimorfismo permite que objetos de diferentes tipos se comportem de maneira semelhante através de uma interface compartilhada."
```

Eu curti, mas ainda deixaria a parte final antes do exemplo para dar um pouco mais de contexto, o que acha?

```suggestion
      "description": "O polimorfismo permite que objetos de diferentes tipos se comportem de maneira semelhante através de uma interface compartilhada, embora cada tipo possa responder de uma forma própria."
```

---

acho uma boa ideia, responde uma possivel duvida que possa surgir ao ler sobre polimorfismo